frontend/src/css/styles.css

```diff
@@ -152,7 +152,7 @@ main .spinner .bounce2 {
 /* PREVIEWER */
 
 #previewer {
-  background-color: rgba(0, 0, 0, 0.9);
+  background-color: rgba(0, 0, 0, 0.99);
```

Comment:
if it's 0.9, we can see the listing in the background. It looks a little bit messy.